source/Ship.cpp

```diff
@@ -133,14 +132,34 @@ namespace {
 
 
 // Construct and Load() at the same time.
-Ship::Ship(const DataNode &node)
-{
-	Load(node);
-}
-
-
-
-void Ship::Load(const DataNode &node)
+Ship::Ship(const DataNode &node,
+		const Set<Effect> &effectsSet,
+		const Set<Outfit> &outfitsSet,
+		const Set<Planet> &planetsSet,
+		const Set<Ship> &shipsSet,
+		const Set<System> &systemsSet,
+		const Government *playerGov
+	)
+{
+	Load(node,
+		effectsSet,
+		outfitsSet,
+		planetsSet,
+		shipsSet,
+		systemsSet,
+		playerGov);
+}
+
+
+
+void Ship::Load(const DataNode &node,
+		const Set<Effect> &effectsSet,
+		const Set<Outfit> &outfitsSet,
+		const Set<Planet> &planetsSet,
+		const Set<Ship> &shipsSet,
+		const Set<System> &systemsSet,
```

Comment:
We need the suffix to avoid the this->prefixing (and mostly to avoid situations where we need the prefixing and forget to add it, causing the code to refer to the wrong variables).